eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/ImageConverter.php

```diff
@@ -176,6 +179,32 @@ public function toFieldValue(StorageFieldValue $value, FieldValue $fieldValue)
         $fieldValue->data = $this->parseLegacyXml($value->dataText);
     }
 
+    public function toStorageFieldDefinition(FieldDefinition $fieldDef, StorageFieldDefinition $storageDef): void
+    {
+        $validators = $fieldDef->fieldTypeConstraints->validators;
+
+        $storageDef->dataInt1 = $validators['FileSizeValidator']['maxFileSize'] ?? 0;
+        $storageDef->dataInt2 = (int)($validators['AlternativeTextValidator']['required'] ?? 0);
+    }
+
+    public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefinition $fieldDef): void
```

Comment:
As a general remark, those "to*" methods would probably be better if they could return the `FieldDefinition` and `StorageFieldDefinition` respectively. I'm not sure what context they are used in, but I think usually those objects are constructed just before this method and are mutated here?

I.e.
```php
    public function toFieldDefinition(StorageFieldDefinition $storageDef, ?FieldDefintion $fieldDef = null): FieldDefinition
    {
        $fieldDef = $fieldDef ?? new FieldDefinition();
        // ...
        return $fieldDef;
    }
```

I understand that this is part of some sort of an interface, so only wanted to comment on it, nothing more.